lib/glific/communications/message.ex

```diff
@@ -206,17 +211,10 @@ defmodule Glific.Communications.Message do
   end
 
   @spec do_receive_message(Contact.t(), map(), atom()) :: :ok | {:error, String.t()}
-  defp do_receive_message(contact, %{organization_id: organization_id} = message_params, type) do
+  defp do_receive_message(contact, %{organization_id: _organization_id} = message_params, type) do
```

Comment:
if we are not using %{organization_id: _organization_id} then just use `message_params`

```
  defp do_receive_message(contact, message_params, type) do
  ```